screed/tests/test_fasta.py

```diff
@@ -145,3 +145,10 @@ class FakeRecord(object):
     w.consume(read_iter)
 
     assert fp.getvalue() == '>foo bar\nATCG\n'
+
+def test_slicing():
+    index = screed.open('screed/tests/test.fa')
```

Comment:
@mr-c Is this an appropriate way to open a file for a nosetest or should I use a 
`self._testfa = os.path.join(os.path.dirname(__file__), 'test.fa')`?